### ISSUE TITLE
[Achievement] Hodir Achievement Accounted for

### ISSUE THREAD

**Issue by Vraskme** [dalaranwow/dalaran-wow#4844]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: After killing Hodir in Ulduar the achievement did not show up as done after our raid killed him. There was a server crash or reset shortly after killing boss as we were distributing loot. Came back onto the server and the loot was gone along with Hodir.

**Expected behaviour**: The Keepers of Ulduar achievement, or at least the kill confirmation check box in that achievement should be shown as completed after he died. The door behind Hodir should have remained open. 

**Steps to reproduce the problem**:

1. Raid Ulduar
2. Kill boss
3. Server restart shortly after kill

**Include proofs for this behaviour**

Proof is shown with the achievement for another mechanic in the fight was achieved but the kill confirmed was not. Screenshot shows Hodir achievement, along with other confirmed kills during the raid.
![screenshot](https://user-images.githubusercontent.com/30773206/29003530-773783c0-7a86-11e7-89b6-d27c8e802c72.png)
![screenshot2](https://user-images.githubusercontent.com/30773206/29003589-64263dc0-7a87-11e7-842e-699686b18890.png)
This is the shot of Hodir dead and gone, but the door is still locked.
![wowscrnshot_080817_193058](https://user-images.githubusercontent.com/30773206/29175099-22918cca-7db6-11e7-8c38-b08d588e49db.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills** 

Raid ID Tag number is 651, the raid was run on August 5th/6th into the morning hours. 
Achievement is Keepers of Ulduar.

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
